cargo-dist/src/announce.rs

```diff
@@ -503,6 +503,18 @@ pub fn announcement_github(manifest: &mut DistManifest) {
             }
             writeln!(&mut gh_body).unwrap();
         }
+
+        if !other_artifacts.is_empty() && manifest.github_attestations {
+            writeln!(&mut gh_body, "## Verifying GitHub Artifact Attestations\n",).unwrap();
+            writeln!(&mut gh_body, "The artifacts in this release have attestations generated with GitHub Artifact Attestations. These can be verified by using the [GitHub CLI](https://cli.github.com/manual/gh_attestation_verify):").unwrap();
+            // TODO: How do we get the repo owner (org) and repo names to probide them in this example?
```

Comment:
Thanks, I'll use that! Yeah, I'm looking for the source repo as that's where the attestations will be stored since the builds happen there.

I didn't see the new target repo feature! Thanks for pointing that out so I can avoid confusing myself 😅

Reply:
Please ignore until Monday lol, but I see I only have access to the `DistManifest` in the announce module when building up the extra section. I'm not sure if adding a field for the source repo there would make sense. Was there another way of getting that inherited repo pair? (I could add it to the GithubCIInfo struct but I guess there's a more generic place for the org/repo pair in the DistManifest. I think I can only get the repo release urls. I obviously misunderstood what you referred to here after digging in. Sorry! 😅

Reply:
hey @dunxen ! sorry for the delay in response and for the bas original advice this is totally my bad. i thought we already had this in the distmanifest, and the answer is that we Sort Of Do (by proxy, in `artifacts_download_url`). but this is not the greatest spot, and would require that you parse it. do you have a thought on where it might go? i definitely think we can and should add both the source repo and source owner as strings somewhere. (i'll think on the location, but if you want to make a proposal, we're open!)

Reply:
yeah i definitely do not think we should make it configurable. parsing can be a pain and so the separate strings seem convenient to me, but i don't have a strong opinion. @mistydemeo any thoughts?